common/source/docs/common-heewingf405.rst

```diff
@@ -0,0 +1,166 @@
+   -  2S - 6S Lipo PMU unit providing:
+   -  88A Current monitor
+   -  9V/3A BEC for powering Video Transmitter
+   -  5V, 5A BEC for internal,servos, and peripherals
+
+-  **Interfaces**
+
+   -  9x PWM outputs DShot capable
+   -  1x RC input/ can be used as full UART with :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` = 1
+   -  4x UARTs/serial for GPS and other peripherals
+   -  I2C port for external compass, airspeed, etc.
+   -  USB-C port on remote dongle
+
+.. note:: the original version has no logging capability. The V2 version has a microSD card for logging, etc.
+
+-  **Size and Dimensions**
+
+   - 44.3mm x 34.3mm x 13.7mm
+   - ??g
+
+Where to Buy
+============
+
+`HEEWING <https://www.heewing.com/>`__
+
+User Manual
+===========
+
+`User Manual <https://cdn.shopifycdn.net/s/files/1/0553/6573/0348/files/fx-405.pdf>`__
+
+Pinouts
+=======
+
+.. image:: ../../../images/heewingf405-pinout.jpg
+    :target: ../_images/heewingf405-pinout.jpg
+
+.. note:: servo/motor power rail supplied from 5V PMU input. Disconnect any ESC BEC's power lead
+
+Default UART order
+==================
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ 
+- SERIAL0 -> USB
+- SERIAL1 -> UART3 (marked UART1 on casing, DMA-enabled)
+- SERIAL2 -> UART1 (marked UART2 on casing, DMA-enabled)
+- SERIAL3 -> UART5 (GPS)
+- SERIAL4 -> UART4 (not pinned out)
+- SERIAL5 -> UART6 (TX only on VTX connector)
+- SERIAL6 -> UART2 (RCIN RX-only or RX/TX with :ref:`BRD_ALT_CONFIG <BRD_ALT_CONFIG>` = 1, DMA-enabled)
+
+Serial protocols shown are defaults, but can be adjusted to personal preferences.
+
+Servo/Motor Outputs
+===================
+
+The HEEWING F405 supports up to 9 PWM outputs. M1-M2, M5-M6 and M9 support bi-directional dshot. All outputs support dshot although M3-4 are not recommended for this purpose as they share resources with I2C.
+
+The PWM is in 4 groups:
+
+- PWM 1 and 2 in group1
+- PWM 3 and 4 in group2
+- PWM 5 and 6 in group3
+- PWM 7-9 in group4
+
+Channels within the same group need to use the same output rate. If any channel in a group uses DShot then all channels in the group need to use DShot.
+
+RC Input
+========
+
+The RCIN pin, is passed by an inverter to RX2 (UART2 RX), which by default is mapped to a timer input instead of the UART, and can be used for all ArduPilot supported receiver protocols, except CRSF/ELRS and SRXL2 which require a true UART connection. However, FPort, when connected in this manner, can provide RC without telemetry. 
+
+To allow CRSF and embedded telemetry available in Fport, CRSF, and SRXL2 receivers, the RX2 pin can also be configured to be used as true UART2 RX pin for use with bi-directional systems by setting the :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to “1” so it becomes the SERIAL2 port's RX input pin.
+
+With this option, :ref:`SERIAL6_PROTOCOL<SERIAL2_PROTOCOL>` must be set to "23", and:
+
+- PPM/SBUS/DSM/SRXL is not supported.
```

Comment:
It's only PPM/SBUS - DSM and SRXL are non-inverted so should work fine

Reply:
will change